### DISCUSSION TITLE
How to close the app

### DISCUSSION THREAD

**Discussion by Pete-Swanson** [MonitorControl/MonitorControl#1652] [Q&A]
Hi, 

Is there a straightforward way to kill the app/turn it off? Right now the only two ways I am able to kill the process is to either restart my mac or run 
`ps aux | grep Monitor`
then grab the PID and run 
`kill -9 <the PID>`

Am I missing an obvious way to turn this off?

Thanks

**Comment by waydabber** [comment]
`killall MonitorControl` should work as well.

(Or you can click on the Quit option in the app menu obviously.)
